test/e2e/ginkgo_operator-sdk.go

```diff
@@ -0,0 +1,230 @@
+package e2e
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/redhat-developer/build/pkg/apis"
+	operator "github.com/redhat-developer/build/pkg/apis/build/v1alpha1"
+	pipelinev1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+
+	. "github.com/onsi/ginkgo"
+)
+
+// Logf logs data
+func Logf(format string, args ...interface{}) {
+	currentTime := time.Now().UTC().Format(time.RFC3339)
+	ginkgoNode := getGinkgoNode()
```

Comment:
can u avoid adding a new var, maybe use `getGinkgoNode()` directly in the printf.

Reply:
Done, see https://github.com/redhat-developer/build/pull/163/commits/7ee9f0765f5f92753d685a0c5ca7efed0be5665d#diff-7898085ea5244b6291d0ad9e253a1191R27